src/pages/Login/index.jsx

```diff
@@ -1,15 +1,35 @@
+
 import React from "react";
 import { Link } from "react-router-dom";
+import Logo from "../../components/logo";
+import Container from "../../components/main";
+import Footer from "../../components/footer.js";
 
 const Login = () => {
-  return (
-    <div>
-      <h1>ESSA É A LOGIN</h1>
-      <p>
-        <Link to="/Register"> VOLTE PARA O REGISTER</Link>
-      </p>
-    </div>
-  );
+  return <>
+    <Container>
+       <div className="inputs-container">
+       <Logo />
+        <form>
+          <label for="login">Login:
```

Comment:
Aqui nem precisava usar o for="login", ou se usar, ele usa o ID como referência e o input tá sem ID.